dbt_project.yml

```diff
@@ -0,0 +1,41 @@
+name: 'snowplow_media_player'
+version: '0.1.0'
+config-version: 2
+require-dbt-version: ">=1.0.0"
+
+profile: 'default'
+
+model-paths: ["models"]
+analysis-paths: ["analyses"]
+test-paths: ["tests"]
+seed-paths: ["seeds"]
+macro-paths: ["macros"]
+docs-paths: ["docs"]
+snapshot-paths: ["snapshots"]
+
+target-path: "target"
+clean-targets:
+  - "target"
+  - "dbt_packages"
+
+vars:
+  snowplow__percent_progress_boundaries: [10, 25, 50, 75]
+  snowplow__valid_play_sec: 30
+  snowplow__complete_play_rate: 0.99
+  snowplow__max_media_pv_window: 10
+
+models:
+  snowplow_media_player:
+    +bind: false
+    +materialized: view
+    web:
+      +schema: "derived"
+      +tags: "snowplow_web_incremental"
+      +enabled: true
+      scratch:
+        +schema: "scratch"
+        +tags: "scratch"
+    custom:
+      +schema: "scratch"
+      +tags: "snowplow_web_incremental"
+      +enabled: false
```

Comment:
This enabled should be triggered by a variable, which should be defined in your vars and in the documentation so that users can easily enable/disable these custom tables, right? Or how else would we expect users to build off of them?

Reply:
Fair enough, although I do think we should change it to being behind a variable once this dbt bug is fixed https://github.com/dbt-labs/dbt-core/issues/3698 to align with the web and mobile packages and their optional modules, but it's good for now!